Arduino Yun/Main/Main.ino

```diff
@@ -0,0 +1,55 @@
+#include <Bridge.h>
+#include <BridgeServer.h>
+#include <BridgeClient.h>
+#include <Servo.h>
+
+BridgeServer server;
+
+void setup() {
+  pinMode(13, OUTPUT);
+  
+  Serial.begin(9600);
+  Bridge.begin();
+
+  server.listenOnLocalhost();
+  server.begin();
+}
+
+void loop() {
+  BridgeClient client = server.accept();
+  if(client){
+    process(client);
+    client.stop();
+  }
+  delay(50);
+}
+
+void process(BridgeClient client){
+  String command = client.readStringUntil('/');
+  command = command.substring(0,command.length()-2);
```

Comment:
To be honest I'm not to sure how the Arduino deals with multiple files. For further iterations I can have a look at how we could refractor the code for the lock to make it more suitable for changing parts in the future. For now I would like to stick to making sure the peripheral actually does what it should do and have all the mechanical parts in place.

Regarding the comment about the indentation. To work with the Arduino I'm kind of forced to use the specialized Arduino IDE. This IDE has no way of setting the indentation to 4 spaces for each tab.